Model/Media.php

```diff
@@ -630,7 +629,7 @@ public function getCategory()
     /**
      * @param CategoryInterface $category|null
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
if I add CategoryInterface I will be doing a BC since it's removing it's not adding a BC this is the difference.